### ISSUE TITLE
[Accessibility] Solve front page text and color contrast issues

### ISSUE THREAD

**Issue by hostle83** [DSpace/dspace-angular#3810]
## Describe the bug
There are a lot of color contrast issues on the front page of the DSpace Demo instance.

![color contrast front page](https://github.com/user-attachments/assets/b26d33d7-04d9-471f-8e5d-e55d36d33b15)

## To Reproduce
Go to the DSpace Demo Site and, using a color contrast tool, such as Silktide, check the color contrasts for the text against the image.

## Expected behavior
Increase color contrast by blurring the image further, by making it darker, and/or by altering the text (font color, font size, style, etc.)

## Related work
Related to https://github.com/DSpace/dspace-angular/issues/3809

**Comment by hostle83** [comment]
Good afternoon, @Andrea-Guevara! 

Thank you very much for your message. This might just be me, but I am unable to see a difference between the image you shared and the screenshot in the initial bug report above. 

Would you please be so kind and point me in the right direction here? Thank you!

**Comment by Andrea-Guevara** [comment]
Good morning @hostle83! I understand, it's a bit hard to tell the difference. We've made a pull available for you to test. If necessary, we're happy to change anything.

If this change still isn't good enough, one thing we could do is increase the font size!

**Comment by hostle83** [comment]
Hi @Andrea-Guevara, thank you! I was able to check out the PR and test it and it looks much better indeed!  

However, I am still receiving color contrast issues when running accessibility checks in Silktide. With your PR, the contrast improves from 3.36:1 to 3.95:1. The goal is to reach 4.5:1. 

I use Silktide to get an accurate measure of color contrast. It's a free [Chrome extension](https://chromewebstore.google.com/detail/silktide-accessibility-ch/mpobacholfblmnpnfbiomjkecoojakah?hl=en&pli=1). 
Once you have added it to Chrome and have activated it, click on "Accessibility Checker" --> "Text contrast" to get a detailed overview of text that fails the contrast ratio test. 

Please let me know if you have any questions!

**Comment by Andrea-Guevara** [comment]
Good afternoon @hostle83! I was able to use the accessibility tool you are using. I think it's for real now. The green color of the text was not in good contrast so I lightened it trying to keep the shade of green a bit. We're open to suggestions!
